edgelet/iotedged/src/lib.rs

```diff
@@ -297,6 +302,33 @@ impl Main {
     }
 }
 
+fn prepare_workload_ca<C>(crypto: &C) -> Result<(), Error>
+where
+    C: CreateCertificate,
+{
+    let edgelet_ca_props = CertificateProperties::new(
+        IOTEDGED_VALIDITY,
+        IOTEDGED_COMMONNAME.to_string(),
+        CertificateType::Ca,
+        IOTEDGED_CA_ALIAS.to_string(),
+    ).with_issuer(CertificateIssuer::DeviceCa);
+
+    let _workload_ca_cert = crypto
```

Comment:
>let _workload_ca_cert [](start = 4, length = 21)

Just curious. Do you get a warning here if you don't assign to a variable? If not, i would remove it since you don't need.

Reply:
Sure will fix.